azkaban-common/src/main/java/azkaban/executor/ExecutionReference.java

```diff
@@ -19,6 +19,7 @@
 public class ExecutionReference {
   private final int execId;
   private Executor executor;
+  //Todo jamiesjc: deprecate updateTime in ExecutionReference class gradually.
   private long updateTime;
```

Comment:
In this PR, I'm only making the change not to read updateTime from active_executing_flows table when fetching active/queued flows. Plan to do the actual removal in a separate PR.